service/collector/tiller_unreachable.go

```diff
@@ -0,0 +1,42 @@
+package collector
+
+import (
+	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	errorLabel = "error"
+)
+
+var (
+	tillerUnreachableDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(Namespace, "", "tiller_unreachable"),
```

Comment:
i would invert this, actually. so something like

`chart_operator_tiller_up`, with a 1 if it is reachable, and a 0 if it's down

Reply:
Done, instead of `tiller_up` I've named it `tiller_reachable`, this metric tells us about the state of tiller from chart-operator's point of view. I've also taken the opportunity to remove the error label, PTAL.